docs/Using Fleet/Usage-statistics.md

```diff
@@ -25,6 +25,10 @@ Below is the JSON payload that is sent to Fleet Device Management Inc:
   "hostExpiryEnabled": true,
   "mdmWindowsEnabled": false,
   "liveQueryDisabled": false,
+  "aiFeaturesDisabled": false,
+  "maintenanceWindowsEnabled": true,
```

Comment:
### Dev note

Whether at least one team has `integrations.google_calendar.enable_calendar_events` set to true